MvvmCross.Android.Support/Fragment/MvxCachingFragmentStatePagerAdapter.cs

```diff
@@ -92,18 +90,5 @@ private static IMvxViewModel GetViewModel(MvxViewPagerFragmentInfo fragmentInfo)
 
             return viewModelLoader.LoadViewModel(fragmentInfo.Request, null);
         }
-        
-        private static Bundle GetArguments(MvxViewPagerFragmentInfo fragmentInfo)
```

Comment:
Unfortunately, I did not see your PR when I made my #3549. I would like to set `Arguments` as this allows to get `MvxViewModelRequest` inside the fragment.